filters/validations.py

```diff
@@ -118,3 +119,73 @@ def fn(value):
                 '<{0}> is not a valid csv of integers'.format(value)
             )
     return fn
+
+
+
+class GenericSeperatedValidator(object):
+    '''
+    Creates list like validator for any voluptuous validation function
+    and any custom string seperator.
+    GenericSeperatedValidator.validation() should be passed to the schema
+
+    >>> CSVofIntegers = GenericSeperatedValidator(int, ',')
+    >>> CSVofIntegers.validation()('1,2,3')
```

Comment:
Instead of naming the method `validation`, you can override the `__call__` method so that this validator complies with the others.